src/Pickles/Pickles.BaseDhtmlFiles/Index.html

```diff
@@ -195,6 +195,9 @@ <h4 data-bind="text: Name, click: $root.toggleDetail" class="canHighlight, click
 
                                 </div>
 
+                                <a class="scenario-link clickable" data-bind="ifnot: $root.disableScenarioLinks, click: $root.getLink" title="Get Link for Scenario">
```

Comment:
Notice the `disableScenarioLinks` property on the vm?  Scenario links only work if you are serving the Pickles page via HTTP.  If its coming from the file system (`file:\\`), deep-linking is not supported.  This is because browser security disables the push-state feature I need to make the linking work when served from the file system.